Update linux prereqs owner

Thraka opened the pull request:
- As discussed with @leecow, changed ownership of this article.
- Dropped preview info
- I've updated the tables to match what is on the release notes for 3.0, 2.2, 2.1
- I removed 1.x as it's no longer supported.
- Filed issue #14675 to track work in the Oct sprint to rewrite this content and change all ownership over to product unit.

Fixes #14217 
Fixes #14655

Thraka committed:
Update linux prereqs owner

- Changed owner to @leecow 
- Updated Preview 1 info to Preview 9

Thraka committed:
Forgot author

mairaw commented:
To update the ms.date, I feel like we should make sure the entire article is correct. See some questions around this here: https://github.com/dotnet/core/pull/3339

Thraka commented on docs/core/linux-prerequisites.md:
```diff
@@ -1,9 +1,9 @@
-ms.date: 12/14/2018
+author: leecow
+ms.author: leecow
+ms.date: 09/09/2019
```
```suggestion
ms.date: 12/14/2018
```

Thraka committed:
Update docs/core/linux-prerequisites.md

Revert date

Thraka commented:
Ignoring the warning about the tab name for now.

Thraka committed:
Update docs/core/linux-prerequisites.md

mairaw commented:
I've missed this PR before. Can you update it for RTM now?

Thraka commented on docs/core/linux-prerequisites.md:
```diff
@@ -55,24 +55,24 @@ For download links and more information, see [.NET Core 1.1 downloads](https://d
+# [.NET Core 3.0](#tab/netcore30)
 
-.NET Core 3.0 Preview 1 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions. 
+.NET Core 3.0 Preview 9 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions. 
```
```suggestion
.NET Core 3.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions. 
```

Thraka commented on docs/core/linux-prerequisites.md:
```diff
@@ -55,24 +55,24 @@ For download links and more information, see [.NET Core 1.1 downloads](https://d
 For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
 
-.NET Core 3.0 Preview 1 is supported on the following Linux distributions/versions. 
+.NET Core 3.0 Preview 9 is supported on the following Linux distributions/versions. 
```
```suggestion
.NET Core 3.0 is supported on the following Linux distributions/versions. 
```

Thraka committed:
Apply suggestions from code review

Thraka committed:
Moving core 3 to top; drop preview name

Thraka committed:
update windows prereqs

Thraka committed:
move core 3 to top of list in macos prereqs

Thraka committed:
Merge branch 'master' of github.com:dotnet/docs into thraka-patch-1

Thraka committed:
Merge branch 'Thraka-patch-1' of github.com:dotnet/docs into thraka-patch-1

Thraka commented:
@mairaw I tidied up the other articles a bit too. I moved core 3 to the front of the tabs too.

Thraka committed:
Reset windows article

Thraka committed:
Merge branch 'master' of github.com:dotnet/docs into Thraka-patch-1

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,6 +19,32 @@ This article shows the dependencies needed to develop .NET Core applications on
+
+OS                            | Version               | Architectures  
+------------------------------|-----------------------|----------------
+Red Hat Enterprise Linux      | 6                     | x64
```
Looks weird to break down the version into two different rows for the same OS. 
```suggestion
Red Hat Enterprise Linux      | 6, 7                  | x64
```

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,6 +19,32 @@ This article shows the dependencies needed to develop .NET Core applications on
+Red Hat Enterprise Linux<br>CentOS<br>Oracle Linux  | 7                     | x64
+Fedora                        | 28, 29, 30            | x64
+Debian                        | 9, 10                 | x64, ARM32\*, ARM64\*
+Ubuntu                        | 16.04+, 18.04+, 19.04+ | x64, ARM32\*, ARM64\*
```
If you're going to use the + notation, instead of "or later versions", you need to explain what the + means before the table. Not all international customers understand that concept.

Thraka replied:
There are no more + in the article, so this one is resolved. :)

mairaw reviewed (changes requested):
LGTM but left a few suggestions for the Linux article

Thraka committed:
Apply suggestions from code review

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

Thraka committed:
Update table for linux prereqs

Thraka committed:
Fix version

mairaw commented:
There are still two comments that weren't addressed:

- https://github.com/dotnet/docs/pull/14315#discussion_r327366364
- https://github.com/dotnet/docs/pull/14315#discussion_r327365136

Thraka commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,6 +19,32 @@ This article shows the dependencies needed to develop .NET Core applications on
+SUSE Enterprise Linux (SLES)  | 12 SP2+, 15           | x64
+Alpine Linux                  | 3.8+                  | x64, ARM64
+
+\* ARM32 and ARM64 support starts with Debian 9 and Ubuntu 16.04. Earlier versions of those distros are not supported on ARM chips.
```
Sorry I thought you meant other prereqs topics, which don't have tables. I'll update them

Thraka committed:
Converted all core release bullets to tables

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
+| CentOS                         | 7                     | x64 |
+| Fedora                         | 28, 29, 30            | x64 |
+| Debian                         | 9, 10                 | x64, ARM32, ARM64 |
+| Ubuntu                         | 16.04 or later, 18.04 or later, 19.04 or later | x64, ARM32, ARM64 |
```
https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md only lists 16.04+, should we simplify this?

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
+| Fedora                         | 28, 29, 30            | x64 |
+| Debian                         | 9, 10                 | x64, ARM32, ARM64 |
+| Ubuntu                         | 16.04 or later, 18.04 or later, 19.04 or later | x64, ARM32, ARM64 |
+| Linux Mint                     | 18                    | x64 |
```
```suggestion
| Linux Mint                     | 18 and later          | x64 |
```

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
+| Red Hat Enterprise Linux       | 6, 7                  | x64 |
+| Oracle Linux                   | 7                     | x64 |
+| CentOS                         | 7                     | x64 |
+| Fedora                         | 28, 29, 30            | x64 |
```
```suggestion
| Fedora                         | 29 and later            | x64 |
```

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
+| Debian                         | 9, 10                 | x64, ARM32, ARM64 |
+| Ubuntu                         | 16.04 or later, 18.04 or later, 19.04 or later | x64, ARM32, ARM64 |
+| Linux Mint                     | 18                    | x64 |
+| OpenSUSE Leap                  | 42.3 or later         | x64 |
```
This is not listed in https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md. Instead it lists openSUSE 15+

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
+| Ubuntu                         | 16.04 or later, 18.04 or later, 19.04 or later | x64, ARM32, ARM64 |
+| Linux Mint                     | 18                    | x64 |
+| OpenSUSE Leap                  | 42.3 or later         | x64 |
+| SUSE Enterprise Linux (SLES)   | 12 SP2 or later, 15   | x64 |
```
```suggestion
| SUSE Enterprise Linux (SLES)   | 12 SP2 or later       | x64 |
```

mairaw commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
-* Alpine Linux 3.7 or later versions - 64-bit (`x86_64` or `amd64`)
+| OS                             |  Version                |  Architectures   |
+| ------------------------------ | ----------------------- | ---------------- |
+| Red Hat Enterprise Linux       |  6, 7                   | x64 |
```
8 is also supported

```suggestion
| Red Hat Enterprise Linux       |  6 and later            | x64 |
```

mairaw reviewed (dismissed):
I've left some comments on some outdated info on the version tables. Can you continue to check the data and see if everything else is up to date? You can see where I stopped.

Thraka commented:
@mairaw this content was originally gathered at preview 1, I'll update it to match RTM.

Thraka committed:
Updated linux prereqs table to match source repo

Thraka commented:
@mairaw 

- I've updated the tables to match what is on the release notes for 3.0, 2.2, 2.1
- I removed 1.x as it's no longer supported.
- Filed issue #14675 to track work in the Oct sprint to rewrite this content and change all ownership over to product unit.

Thraka committed:
Removed 1.x for all OS; Promoted 3.0 to top of tabs

rpetrusha commented on docs/core/linux-prerequisites.md:
```diff
@@ -19,69 +19,82 @@ This article shows the dependencies needed to develop .NET Core applications on
+See [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md) for the complete list of .NET Core 2.1 supported operating systems, distributions and versions, out of support OS versions, and lifecycle policy links.
 
 ---
 
```
Should the two references to versions earlier than .NET Core 2.1 be removed? The versions are EOL.

rpetrusha commented on docs/core/macos-prerequisites.md:
```diff
@@ -45,32 +24,30 @@ See [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/ma
 For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
 
----
+# [.NET Core 2.x](#tab/netcore2x)
```
2.1 is EOL. Perhaps change this to tab to **.NET Core 2.2:** and remove links to .NET Core 2.1 information.

KalleOlaviNiemitalo replied:
Is 2.1 really EOL? <https://dotnet.microsoft.com/platform/support/policy/dotnet-core> says .NET Core 2.2 end of support will be on December 23, 2019, but .NET Core 2.1 will last "At least three years from LTS declaration (August 21, 2018)".

Thraka replied:
@KalleOlaviNiemitalo no it was just a mistake! 😄

rpetrusha commented on docs/core/macos-prerequisites.md:
```diff
@@ -45,32 +24,30 @@ See [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/ma
 
+# [.NET Core 2.x](#tab/netcore2x)
+
+Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.1) topic for the version you have installed.
```
.NET Core 2.1 is EOL, but the known issues link goes to its known issues, not .NET Core 2.2.

Thraka replied:
@rpetrusha Hey Ron. 2.1 is in LTS, not EOL. 2.0 is EOL though. https://github.com/dotnet/core/blob/master/microsoft-support.md

Yes?

rpetrusha replied:
It's still a better experience to link to a specific .NET Core 2.x download page instead of <https://dotnet.microsoft.com/download>, which is a .NET Core 3.0 page with an option to go to a page with other versions. And if you're covering both 2.1 and 2.2, there should be two separate known issues links, one for each version.

rpetrusha reviewed (changes requested):
I've left three comments about references to EOL versions, but expanding on them, I'm wondering whether in the Windows topic in particular, but also both the macOS and Linux topics, we should remove references to EOL versions, which are extensive. In the Windows topic in particular, there's a disconnect between the version tabs (3.0, 2.x) and the versions listed (all) which can be eliminated by removing references to EOL versions and changing the **.NET Core 2.x** tab in the Windows topic to **.NET Core 2.2**. It seems to me that this topic should focus on OS prerequisites for supported .NET Core versions, not historical information about current and unsupported versions.

rpetrusha commented on docs/core/macos-prerequisites.md:
```diff
@@ -45,32 +24,30 @@ See [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/ma
 
+# [.NET Core 2.x](#tab/netcore2x)
+
+Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.1) topic for the version you have installed.
```
It would be better here to link to one of the installation pages that includes .NET Core 2.2 on it, rather than to a page that requires selecting **All .NET Core downloads..**.

Thraka commented:
@rpetrusha Thanks for reviewing. I want to point out that I have #14675 filed to completely rewrite all 3 of these articles to focus on what you pointed out, prerequisites and less on supported/unsupported version info. Additionally, I added a response about how 2.1 and 2.2 are still supported but 2.0 is EOL.

rpetrusha commented:
I'll approve and you can merge when you're ready, @thraka. I'd still consider changing the link target and adding the second known issues link, though.

Thraka committed:
Fix up some more prereqs

Thraka commented:
@rpetrusha Thanks ron. I split the sections of macos out ask you suggested and updated the links for download/known issues. it is a cleaner approach. Thanks again.
